src/main/java/io/cdap/dynamicschema/DynamicSchemaTableSink.java

```diff
@@ -14,31 +14,31 @@
  * the License.
  */
 
-package co.cask.dynamicschema;
-
-import co.cask.cdap.api.annotation.Description;
-import co.cask.cdap.api.annotation.Name;
-import co.cask.cdap.api.annotation.Plugin;
-import co.cask.cdap.api.common.Bytes;
-import co.cask.cdap.api.data.batch.Output;
-import co.cask.cdap.api.data.format.StructuredRecord;
-import co.cask.cdap.api.data.schema.Schema;
-import co.cask.cdap.api.dataset.DatasetManagementException;
-import co.cask.cdap.api.dataset.DatasetProperties;
-import co.cask.cdap.api.dataset.lib.KeyValue;
-import co.cask.cdap.api.dataset.table.Put;
-import co.cask.cdap.api.dataset.table.Table;
-import co.cask.cdap.etl.api.Emitter;
-import co.cask.cdap.etl.api.PipelineConfigurer;
-import co.cask.cdap.etl.api.batch.BatchRuntimeContext;
-import co.cask.cdap.etl.api.batch.BatchSink;
-import co.cask.cdap.etl.api.batch.BatchSinkContext;
-import co.cask.dynamicschema.api.Expression;
-import co.cask.dynamicschema.api.ExpressionException;
-import co.cask.dynamicschema.api.ObserverException;
-import co.cask.dynamicschema.api.ValidationException;
-import co.cask.dynamicschema.observer.SchemaObserver;
-import co.cask.dynamicschema.observer.StructuredRecordObserver;
+package io.cdap.dynamicschema;
```

Comment:
this looks ok to me, but how come for python evaluator, you were putting 'plugin' in the package name also?

Reply:
I believe it was `co.cask.hydrator.python*`

Reply:
https://github.com/data-integrations/python-plugins/tree/develop/src/main/java/co/cask/hydrator/python/transform